earn/src/components/borrow/BorrowGraph.tsx

```diff
@@ -90,11 +109,31 @@ export default function BorrowGraph(props: BorrowGraphProps) {
   const { graphData } = props;
 
   const now = Date.now();
-  const filteredGraphData = graphData
-    .filter((item) => now - item.x.getTime() <= MILLIS_PER_WEEK)
-    .map((item) => {
-      return { ...item, x: item.x.toISOString() };
-    });
+
+  const sortedGraphData = graphData.sort((item) => item.x.getTime());
+  const startIdx = sortedGraphData.findIndex((item) => now - item.x.getTime() <= MILLIS_PER_DAY * DAYS_TO_SHOW);
```

Comment:
I think we should also check that startIdx != -1 just to be safe. If we choose to not render in this case, we would need to move the useMediaQueryHook up above.